toolchain/semantics/semantics_handle.cpp

```diff
@@ -398,6 +398,21 @@ auto SemanticsHandlePrefixOperator(SemanticsContext& context,
   return context.TODO(parse_node, "HandlePrefixOperator");
 }
 
+auto SemanticsHandleIfExpression(SemanticsContext& context,
```

Comment:
FWIW, I've generally tried to keep these lexically ordered to make it easier to sift through (`IfExpression*` before `IfStatement*`)